conans/test/conftest.py

```diff
@@ -50,45 +57,45 @@
         "0.28": {
             "path": {
                 # Using chocolatey in Windows -> choco install pkgconfiglite --version 0.28
-                'Windows': "C:/ProgramData/chocolatey/lib/pkgconfiglite/tools/pkg-config-lite-0.28-1/bin",
-                'Darwin': f"{homebrew_root}/bin/pkg-config",
-                'Linux': "/usr/bin/pkg-config"
+                'Windows': "C:\\ProgramData\\chocolatey\\lib\\pkgconfiglite\\tools\\pkg-config-lite-0.28-1\\bin",
+                'Darwin': f"{homebrew_root}/bin",
+                'Linux': "/usr/bin"
             }
         }},
     'autotools': {"exe": "autoconf"},
     'cmake': {
         "default": "3.15",
         "3.15": {
-            "path": {'Windows': 'C:/cmake/cmake-3.15.7-win64-x64/bin',
+            "path": {'Windows': 'C:\\cmake\\cmake-3.15.7-win64-x64\\bin',
```

Comment:
Isn't it a bad UX that we Windows users will have to define our ``conftest_user.py`` with double backslash?